### PR TITLE
MAINT: interpolate: delete duplicated FITPACK bisplev interface

### PR DESCRIPTION
**Pull request by AtsushiSakai**
#### Reference issue

Fix a part of #16729

#### What does this implement/fix?

This PR deletes duplicated FITPACK bisplev interface in _fitpackmodule.c.

### CODE REVIEW

File: scipy/interpolate/_fitpack_impl.py
Comment (ev-br) [inline scipy/interpolate/_fitpack_impl.py:670]:
```suggestion
    msg = f"Too many data points to interpolate."
```

**Review by ev-br — APPROVED** [review]
LGTM modulo a small optional suggestion.

**Review by tylerjereddy — COMMENTED** [review]
It has a core developer approval, deletes a bunch of extraneous code, CI is passing, and diff also seems sensible to me as non-expert at least.

I merged Evgeni's small inline suggestion--note that the linter may complain about an f-string where it isn't needed though, not sure. That's easy to fix if it does though..

**Pull request merged**
